robolectric/src/test/java/org/robolectric/util/TestUtil.java

```diff
@@ -206,17 +167,6 @@ public static String stringify(int[] sdk, String manifest, Class<? extends Appli
         "constants=" + constants;
   }
 
-  public static void assertStringsInclude(List<String> list, String... expectedStrings) {
```

Comment:
I think this also (non-obviously) asserts that the strings come in the specified order, which I think we lose with `contains()`, but maybe there's a `containsInOrder()`?